espanol/maquina/tiempo/maquina_del_tiempo.md

```diff
@@ -0,0 +1,2 @@
+Llamada [Gobble-o-tron](https://octodex.github.com/gobbleotron)
```

Comment:
This link seems to be an absolute link to GitHub rather than a relative link.  If you are linking to another page in the create-your-own-adventure repository, please use relative links to do so.  See the README of this repository for more information.